### PR TITLE
fix staticcheck:vendor/k8s.io/kubectl/pkg/cmd/scale

### PR DESCRIPTION
**Pull request by lala123912**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fix staticcheck:vendor/k8s.io/kubectl/pkg/cmd/scale
vendor/k8s.io/kubectl/pkg/cmd/scale/scale.go:212:2: this value of err is never used (SA4006)

**Which issue(s) this PR fixes**:
Part of #92402

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func (o *ScaleOptions) RunScale() error {
 	}
 
 	infos := []*resource.Info{}
-	err = r.Visit(func(info *resource.Info, err error) error {
+	_ = r.Visit(func(info *resource.Info, err error) error {
```
Comment (BenTheElder) [inline staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go:212]: should this error be ignored?

**Review by BenTheElder — COMMENTED** [review]
in the future please have a relevant reviewer review first, I'm not a reviewer / approve in kubectl but I'm happy to approve for hack/
EDIT: I'll approve for hack _after_ the other changes are reviewed + lgtm + approved

**Review by lala123912 — COMMENTED** [review]
> in the future please have a relevant reviewer review first, I'm not a reviewer / approve in kubectl but I'm happy to approve for hack/
> EDIT: I'll approve for hack _after_ the other changes are reviewed + lgtm + approved

ok

**Pull request closed without merging**